mule-extensions-api/src/main/java/org/mule/runtime/extension/internal/loader/enricher/ExtensionTypesDeclarationEnricher.java

```diff
@@ -43,6 +44,8 @@
  */
 public final class ExtensionTypesDeclarationEnricher implements DeclarationEnricher {
 
+  private static String messageTypeId = "org.mule.runtime.api.message.Message";
```

Comment:
Constant should be final and upper case. Anyway, this constant shouldn't exist in the first place. Simply use `Message.class.getName()`

Reply:
fixed